pkg/controller/node/node_controller.go

```diff
@@ -460,6 +460,14 @@ func (ctrl *Controller) updateNode(old, cur interface{}) {
 	}
 	glog.V(4).Infof("Node %s updated", curNode.Name)
 
+	// Let's be verbose when a node changes pool
+	oldPool, err := ctrl.getPrimaryPoolForNode(oldNode)
+	if err == nil {
```

Comment:
Nit: if we don't care about the error, may as well collapse into `if err == nil && oldPool.Name != pool.Name`

Reply:
Fair...though maybe not worth spawning 3 new clusters for that in CI?  I can sneak that change in along with something else?

Reply:
Eh I decided to roll even more logging stuff in anyways.